### PR TITLE
February 2022 Client Updates (packets)

### PR DESCRIPTION
**Pull request by zach2good**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: src/map/packets/inventory_size.cpp [hunk 1]
```diff
@@ -36,7 +36,7 @@ along with this program.  If not, see http://www.gnu.org/licenses/
 CInventorySizePacket::CInventorySizePacket(CCharEntity* PChar)
 {
     this->type = 0x1C;
-    this->size = 0x1A;
+    this->size = 0x64;
```
Comment (claywar) [inline src/map/packets/inventory_size.cpp]: This should use the length() setter, packet size in bytes is 100 (0x64).  If setting directly, this should be halved.

**Comment by zach2good** [comment]
TODO: A migration to update features for existing accounts

**Comment by zach2good** [comment]
https://media.discordapp.net/attachments/883227930828886060/941403150525542430/unknown.png

Seems to be working

**Comment by zach2good** [comment]
I'll handle the migration and the recycle system in another PR

**Review by claywar — APPROVED** [review]

**Pull request merged**